boards/nxp-imx8qm/default.nix

```diff
@@ -0,0 +1,48 @@
+{ config, lib, pkgs, ... }:
+
+let
+  imxOpteeOs = pkgs.callPackage ./imx-optee-os.nix {inherit pkgs;};
+in
+{
+  device = {
+    manufacturer = "NXP";
+    name = "i.MX8QuadMax";
+    identifier = "mx8qm-mek";
+  };
+
+  hardware = {
+    soc = "nxp-imx8qm";
+  };
+
+  Tow-Boot =  {
+
+    uBootVersion = "2022.04";
+
+    src = fetchGit {
+        url = "https://source.codeaurora.org/external/imx/uboot-imx.git";
+        ref = "lf_v2022.04";
+    };
+
+    defconfig = "imx8qm_mek_defconfig";
+
+    useDefaultPatches = true;
+
+    builder = {
+      postPatch = ''
+        install -m 0755 $BL31/bl31.bin ./
```

Comment:
`0644` instead? Especially x-flag with copying all seems overly generous.

Reply:
Thanks, for noticing this! Fixed.